README.md

```diff
@@ -201,4 +213,4 @@ Project Link: [https://github.com/your_username/repo_name](https://github.com/yo
 [license-url]: https://github.com/othneildrew/Best-README-Template/blob/master/LICENSE.txt
 [linkedin-shield]: https://img.shields.io/badge/-LinkedIn-black.svg?style=flat-square&logo=linkedin&colorB=555
 [linkedin-url]: https://linkedin.com/in/othneildrew
-[product-screenshot]: images/screenshot.png
+[product-screenshot]: images/screenshot.png -->
```

Comment:
The `product-screenshot` link is still used, so this one shouldn't be commented out as it breaks the screenshot link.

Also, why not remove the block-links altogether? This duplicates the links which might be confusing.